### ISSUE TITLE
WCAG 4.1.2 test 8: false positive

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#93]
[This document](https://github.com/veraPDF/veraPDF-wcag-algs/files/7396746/test-document.pdf) fails **WCAG 4.1.2 test 8** ('_Invalid table cell_'). As far as I can see, this is a correctly tagged cell in a correctly tagged table. Strangely, this error is only given for this particular cell, and not the other cells of the same table.

Can you check out what causes this apparent false positive?
